contents/blog/best-open-source-session-replay-tools.md

```diff
@@ -0,0 +1,195 @@
+---
+date: 2022-04-07
+title: Open source (and self-hosted) alternatives to Hotjar, FullStory & LogRocket
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-company-culture-blog.png
+featuredImageType: full
+author: ["hanna-crombie"]
+categories: ["Guides", "Product analytics"]
+---
+
+Most analytics tools have one goal in mind: providing actionable insights on user behavior. It's all about hard numbers; views, clicks, conversions. But sometimes all those numbers can feel a bit sterile. Separated from the individual user narrative, statistics are an incomplete picture. Enter session recording.
+
+### Why you need session recording
+
+Session recordings, also known as session replays, are video renderings of actions taken by your users in real-time. They capture mouse movement, clicks, taps, and scrolling across your site or app’s pages.
+
+Aside from making for some fascinating watching, recordings facilitate a detailed investigation of user behavior, allowing you to identify friction areas such as: 
+
+- Where users are getting stuck on your site
+- Hesitation at different stages of your funnel
+- How bugs come about
+- How your users are interacting with individual elements of your product.
+
+This context can help you to optimize user experience, build user-led products and, ultimately, improve conversion and retention. 
+
+Bottom line: session recording is an essential tool for making better products.
+
+### Why self-host your session recording?
+Most popular session recording tools, such as  Hotjar, FullStory and LogRocket, are exclusively cloud-based. This is convenient, but comes with several drawbacks:
+
+1. They're expensive
+2. You don't control your user data
+3. GDPR and HIPAA compliance is a headache 
+
+Self-hosting your session recording eliminates these drawbacks and allows you to keep control of your data, while open source tools give you the freedom to study, modify, and run the tools at no extra cost.
+
+If that sounds good to you, coming up are the best open source (or self-hosted) session recording tools right now.
+
+## 1. [PostHog](https://posthog.com/)
+![PostHog - open source product analytics replay](../images/blog/open-source-hotjar-alternatives/img1_PostHog.png)
+
+PostHog is an all-in-one product analytics platform that offers all the tools required to measure user success and build better products, including but not limited to [session recording](/product/session-recording). Other features include [Feature Flags](/product/feature-flags), [Funnels](/product/funnels), [Heatmaps](/product/heatmaps), [A/B testing](/product/experimentation-suite), [Trends](/product/trends) and [much more](/product). All of PostHog’s tools work together natively, offering extensive, actionable user insights beyond what's possible with standalone session replay tools.
```

Comment:
```suggestion
PostHog is an all-in-one product analytics platform that offers all the tools required to measure user success and build better products, including but not limited to [Session Recording](/product/session-recording). Other features include [Feature Flags](/product/feature-flags), [Funnels](/product/funnels), [Heatmaps](/product/heatmaps), [A/B testing](/product/experimentation-suite), [Trends](/product/trends) and [much more](/product). All of PostHog’s tools work together natively, offering extensive, actionable user insights beyond what's possible with standalone session replay tools.
```

Reply:
We have normally steered away from calling it A/B testing. In the app, it's experimentation.